packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md

```diff
@@ -97,6 +97,35 @@ When using red text for destructive actions, it is still recommended to require
 
 <img src="./img/red-delete-text.png" alt="drilldown menu" width="935"/>
 
+### Disabling menus and menu options
+Menus or menu items can be disabled for multiple reasons. The reason an action is disabled informs which design pattern you should use.
+
+1. **An action can be performed, but is not recommended.**
+When an action can be performed but may result in an undesirable outcome, do not disable it. 
+
+Instead, add a [confirmation model](components/model). When the user clicks on the action, use a modal to explain the potential consequences and ask the user if they are sure they want to proceed.
+
+Example: A user wants to start maintenance on a master node.
+
+2. **The user doesn’t have the correct permissions to perform an action.**
```

Comment:
In this scenario, I would also advise adding a tooltip that says something like "You do not have the required permissions to perform this action. Contact your org admin" 

It would also help to have an image of this here.

Reply:
I feel like this second bullet actually fits into the third - the unmet prerequisite is having the permissions to perform the action... so I would maybe combine them and just add this scenario as an example instead (?)